src/client.ts

```diff
@@ -1176,6 +1187,21 @@ class OptimizelyReactSDKClient implements ReactSDKClient {
       attributes: finalUserAttributes,
     };
   }
+
+  // TODO: discuss if we want to expose these method and provide implementation
+  getVuid(): string | undefined {
+    return undefined;
+  }
```

Comment:
I think @opti-jnguyen is going to move this to `private` in the next 5.0.0 beta I think